src/styles/card.scss

```diff
@@ -0,0 +1,90 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+
+  width: 200px;
+
+  border-radius: 5px;
+  border: 1px solid #F3F3F3;
+  background: $white-color;
+
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 140%;
```

Comment:
no needs to add these styles to the card since you've used font styles for children anyway
```suggestion
```